Alignment/OfflineValidation/plugins/PixelBaryCentreAnalyzer.cc

```diff
@@ -246,322 +246,321 @@ void PixelBaryCentreAnalyzer::analyze(const edm::Event& iEvent, const edm::Event
 
     phase_ = -1;
 
-    const TrackerGeometry* tkGeo = &iSetup.getData(trackerGeometryToken_);
-    const TrackerTopology* tkTopo = &iSetup.getData(trackerTopologyToken_);
+    const TrackerGeometry& tkGeo = iSetup.getData(trackerGeometryToken_);
+    const TrackerTopology& tkTopo = iSetup.getData(trackerTopologyToken_);
 
-    if (tkGeo->isThere(GeomDetEnumerators::PixelBarrel) && tkGeo->isThere(GeomDetEnumerators::PixelEndcap)) {
+    if (tkGeo.isThere(GeomDetEnumerators::PixelBarrel) && tkGeo.isThere(GeomDetEnumerators::PixelEndcap))
       phase_ = 0;
-    } else if (tkGeo->isThere(GeomDetEnumerators::P1PXB) && tkGeo->isThere(GeomDetEnumerators::P1PXEC)) {
+    else if (tkGeo.isThere(GeomDetEnumerators::P1PXB) && tkGeo.isThere(GeomDetEnumerators::P1PXEC))
       phase_ = 1;
-    }
+  }
```

Comment:
there is something funny with parenthesis going on here. I think this is superfluous now and this clause should actually be closed at L545 of the new layout. 

https://github.com/cms-sw/cmssw/blob/393e0cd2e3f3793b56b81b12c06042d1f793c651/Alignment/OfflineValidation/plugins/PixelBaryCentreAnalyzer.cc#L545

This also explains why the formatting of the following piece:

https://github.com/cms-sw/cmssw/blob/393e0cd2e3f3793b56b81b12c06042d1f793c651/Alignment/OfflineValidation/plugins/PixelBaryCentreAnalyzer.cc#L548-L564

is odd.

Reply:
You're right, I decided to fix it in the first commit and rebase